src/pages/MakeLink/MakeLink.tsx

```diff
@@ -109,6 +115,14 @@ const MakeLink = () => {
             helperText={"Peer ID"}
           />
         </Grid>
+        <Grid item={true} xs={12} md={6}>
+          <TextField
+            value={addressHint}
+            onChange={event => setAddressHint(event.target.value)}
+            label={"Peer Address Hint"}
+            helperText={"Hint for the peer address in multiaddress format"}
```

Comment:
That kind of only restates what is in the label just in a longer form.

I think the `helperText` should contain that this address will be dialed in addition to the peer ID. Or something else that adds more information to what this value does.

Be creative :)